Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -266,6 +272,26 @@ public float PointerExtent
         /// <inheritdoc />
         public float SphereCastRadius { get; set; } = 0.1f;
 
+        [SerializeField]
+        [Range(0f, 360f)]
+        [Tooltip("The Y orientation of the pointer - used for touchpad rotation and navigation")]
```

Comment:
could this not be used with thumbstick rotation?

Reply:
It is also used with thumbstick rotation. 

I copied as is. I can update.